src/Intervention/Image/Gd/Commands/ResizeCommand.php

```diff
@@ -62,6 +63,10 @@ protected function modify($image, $dst_x, $dst_y, $src_x, $src_y, $dst_w, $dst_h
             imagesavealpha($modified, true);
         }
 
+        if ($interpolationAlgo && function_exists('imagesetinterpolation')) {
+            imagesetinterpolation($modified, $interpolationAlgo);
+        }
+
```

Comment:
In what cases is it possible to have GD but not this method? 🤔 Just wondering, thanks

Reply:
Btw, `function_exists` in other cases is actually always used to throw an exception and not silently ignore a feature, doesn't sound like a good DX to me.

Reply:
imagesetinterpolation has been added in php 5.5 - since this library supports lower php version ....

I'm fine to throw an exception as well ... ;-)

Reply:
> imagesetinterpolation has been added in php 5.5 - since this library supports lower php version ....

Ah, nice!

I think the exception definitely makes more sense then quietly ignoring it => however unlikely.